src/4.6/en/writing-tests-for-phpunit.xml

```diff
@@ -572,7 +572,7 @@ class ExceptionTest extends PHPUnit_Framework_TestCase
 
     /**
      * @expectedException              InvalidArgumentException
-     * @expectedExceptionMessageRegExp /Right.*/
+     * @expectedExceptionMessageRegExp /Right.* /
```

Comment:
`/` actually is the regular expression delimiter, it can be any other char, people usually use `@` or `#` too, but it can be any other char.